lms/djangoapps/program_enrollments/tests/test_signals.py

```diff
@@ -0,0 +1,79 @@
+"""
+Test signal handlers for program_enrollments
+"""
+
+from __future__ import absolute_import
+
+from lms.djangoapps.program_enrollments.api.v1.tests.factories import ProgramEnrollmentFactory
+from lms.djangoapps.program_enrollments.signals import _listen_for_lms_retire
+from openedx.core.djangoapps.user_api.accounts.tests.retirement_helpers import fake_completed_retirement
+from student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+
+class SurveyRetireSignalTests(ModuleStoreTestCase):
```

Comment:
I never saw this before, but..."Survey"?

Reply:
I totally missed that too. I coped these tests from the survey answers retirement tests